wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Content/Rich Content/ConversationLocationMessageCell.swift

```diff
@@ -179,37 +176,6 @@ final class ConversationLocationMessageCell: UIView, ConversationMessageCell, Co
     private func openInMaps() {
         lastConfiguration?.location.openInMaps(with: mapView.region.span)
     }
-
-}
-
-// MARK: - context menu
-extension ConversationLocationMessageCell: UIContextMenuInteractionDelegate {
-
-    func contextMenuInteraction(_ interaction: UIContextMenuInteraction,
```

Comment:
Have you moved this code to another place?

Reply:
no, if no menu interaction is set then it's using implementation with new context menu